### ISSUE TITLE
iPhone XS, XR, and XS Max

### ISSUE THREAD

**Issue by patkozlowski** [dannycabrera/Get-iOS-Model#13]
Are you planning on supporting these the new devices:  XS, XR, and XS Max.  That would be great.

**Comment by dannycabrera** [comment]
Yes, have it coded and was waiting for Apple to officially release the hardware string. Just checked and it’s available so will update both GitHub and nuget today. Thanks for asking.

**Comment by patkozlowski** [comment]
Thank you Danny for a lightning-fast response and turnaround 👍
